INSTALL.md

```diff
@@ -141,65 +141,67 @@ packages necessary for building .debs, so be prepared to type your password for
 
 <a name="osx" />
 ##OS X
+Before we start, be aware that this section had a large guide that succeeded in only confusing.
```

Comment:
This line is really not necessary. It's not clear why users should be aware what you think of the previous version.